Src/Fido2/Fido2NetLib.cs

```diff
@@ -140,11 +144,11 @@ public CredentialMakeResult(string status, string errorMessage, AttestationVerif
     /// </summary>
     /// <param name="credentialIdUserParams"></param>
     /// <returns></returns>
-    public delegate Task<bool> IsCredentialIdUniqueToUserAsyncDelegate(IsCredentialIdUniqueToUserParams credentialIdUserParams);
+    public delegate Task<bool> IsCredentialIdUniqueToUserAsyncDelegate(IsCredentialIdUniqueToUserParams credentialIdUserParams, CancellationToken cancellationToken);
```

Comment:
Since this is an public API we want to make the CancellationToken optional.

Reply:
The same as https://github.com/passwordless-lib/fido2-net-lib/pull/278#discussion_r782596266